JitsiMeetJS.ts

```diff
@@ -90,6 +91,13 @@ interface IJitsiMeetJSOptions {
     }
 }
 
+interface ICreateLocalTrackFromMediaStreamOptions {
+    stream: MediaStream,
+    sourceType: string,
```

Comment:
What is this used for, other than documentation?

Reply:
Purely documentation and typings for the function

Reply:
Or if you mean the sourceType, then it seems to be a way for client apps to determine the type of stream, e.g. https://github.com/jitsi/lib-jitsi-meet/blob/dd81069cb0e18a387c80cae149bfe6f08b52a6c4/modules/proxyconnection/ProxyConnectionService.js#L286

Reply:
Gotcha. We are planning to phase that one out, so I can take of it when the day comes. Thanks for checking!